oat-sa/extension-tao-itemqti
# fix/TR-5000/horizontal-jp-symbols-eliminate-choices

related to https://oat-sa.atlassian.net/browse/TR-5000

### Description 

JP symbol sequence of `除外` is treated as 2 separate words by browser, thus breaking it into 2 lines. 

This fix adds `white-space: nowrap` rule to fix this behaviour

### How to test

 - Create item in authoring with Eliminable choices
 - Set interface lang to ja-JP
 - Check how the label `Eliminable` (`除外`) is displayed
